### PR TITLE
added a readme

### PR DESCRIPTION
**Pull request by jyl1**
I added a readme to your cool repo!

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -0,0 +1 @@
+"is a readme"
```
Comment (PatriciaAzike) [inline README.md]: I see you added a Readme. That's a good practice I need to imbibe.

**Review by PatriciaAzike — CHANGES_REQUESTED** [review]
Add more stuff to the Readme, please.

File: README.md (hunk 1 above)
Comment (jyl1) [reply README.md]: Thx i added more stuff.

**Review by PatriciaAzike — APPROVED** [review]
Thanks. This is really interesting.

**Pull request merged**